core/src/main/scala/zio/logging/package.scala

```diff
@@ -7,6 +7,15 @@ package object logging {
   def log(level: LogLevel)(line: => String): ZIO[Logging, Nothing, Unit] =
     Logging.log(level)(line)
 
+  def error(cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
```

Comment:
To avoid needless name collisions, maybe we should prefix all "package object methods" with log, e.g. `logError`, `logThrowable`, `logLocally`, etc.

Reply:
done